resources/views/dashboard/admin/realops/index.blade.php

```diff
@@ -30,7 +30,7 @@
             <th scope="col">Departure Time (ET)</th>
             <th scope="col">Departure Airport</th>
             <th scope="col">Arrival Airport</th>
-            <th scope="col">Estimated Arrival Time (ET)</th>
+            <th scope="col">Estimated Enroute Time (ET)</th>
```

Comment:
Remove (ET) - that indicated eastern time and isn't applicable to an ETE (duration)

Reply:
Nice catch, will fix.

Reply:
Fixed